ui/components/DashboardComponent.js

```diff
@@ -414,6 +416,19 @@ class DashboardComponent extends React.Component {
     });
   };
 
+  /**
+   * redirectErrorToConsole returns a function which redirects
+   * ther error to the console under the group labelled by the "msg"
+   * param
+   * @param {string} msg 
```

Comment:
Have you configured your editor as typescript one? Why such annotations get added everytime?

Reply:
No, actually they are [JSDoc](https://jsdoc.app/),  I add them manually as they assist:
1. In understanding what code does and helps other developers to understand what parameters a function can take in and what it can return, `any` isn't very helpful.
2. It helps code editor/IDE to facilitate code completion.
3. I use `// @ts-check` to check for type issues before pushing the code to ensure that I am not invoking any invalid method. By default editors use type inference to show such errors but usually they fallback to infer `any` which again isn't very helpful.

Reply:
Nice tool, thanks for info!